src/app/pages/Sandbox/Editor/Content/Header/index.js

```diff
@@ -297,6 +314,7 @@ export default class Header extends React.PureComponent<Props> {
             ) : (
               <Action
                 onClick={userActions.signIn}
+                tooltip="Sign in with Github"
```

Comment:
I think a tooltip here will be redundant as the button has text 🤔

Reply:
Good point, will remove.